File: Examples/VisualBasic/Presentations/ModifyBuiltinProperties.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Slides

Namespace Aspose.Slides.Examples.VisualBasic.Presentations
    Public Class ModifyBuiltinProperties
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Presentations()

            ' Instantiate the Presentation class that represents the Presentation
            Dim pres As New Presentation(dataDir & "ModifyBuiltinProperties.pptx")

            ' Create a reference to IDocumentProperties object associated with Presentation
            Dim dp As IDocumentProperties = pres.DocumentProperties

            ' Set the builtin properties
            dp.Author = "Aspose.Slides for .NET"
            dp.Title = "Modifying Presentation Properties"
            dp.Subject = "Aspose Subject"
            dp.Comments = "Aspose Description"
            dp.Manager = "Aspose Manager"

            ' Save your presentation to a file
            pres.Save(dataDir & "Updated_Document_Properties_out.pptx", Aspose.Slides.Export.SaveFormat.Pptx)

        End Sub
    End Class
End Namespace

File: Examples/CSharp/Presentations/ModifyBuiltinProperties.cs

using System.IO;

using Aspose.Slides;
using Aspose.Slides.Export;

namespace Aspose.Slides.Examples.CSharp.Presentations
{
    public class ModifyBuiltinProperties
    {
        public static void Run()
        {
            // For complete examples and data files, please go to https:// Github.com/aspose-slides/Aspose.Slides-for-.NET

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Presentations();

            // Instantiate the Presentation class that represents the Presentation
            Presentation presentation = new Presentation(dataDir + "ModifyBuiltinProperties.pptx");

            // Create a reference to IDocumentProperties object associated with Presentation
            IDocumentProperties documentProperties = presentation.DocumentProperties;

            // Set the builtin properties
            documentProperties.Author = "Aspose.Slides for .NET";
            documentProperties.Title = "Modifying Presentation Properties";
            documentProperties.Subject = "Aspose Subject";
            documentProperties.Comments = "Aspose Description";
            documentProperties.Manager = "Aspose Manager";

            // Save your presentation to a file
            presentation.Save(dataDir + "DocumentProperties_out.pptx", SaveFormat.Pptx);
            
        }
    }
}